From e0a3caa2206a851a3311bf42ddfb1558314868b1 Mon Sep 17 00:00:00 2001
Message-Id: <e0a3caa2206a851a3311bf42ddfb1558314868b1.1387276076.git.minovotn@redhat.com>
In-Reply-To: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
References: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 10 Dec 2013 15:12:55 +0100
Subject: [PATCH 14/16] trace-events: Clean up with
 scripts/cleanup-trace-events.pl again

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1386688376-29521-7-git-send-email-armbru@redhat.com>
Patchwork-id: 56116
O-Subject: [PATCH 7.0 qemu-kvm 6/7] trace-events: Clean up with scripts/cleanup-trace-events.pl again
Bugzilla: 997832
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

Event qxl_render_blit_guest_primary_initialized is unused since commit
c58c7b9, drop it.

Commit 42e5b4c moved hw/ppc/xics.c to hw/intc/xics.c without updating
the comment in trace-events.

"scripts/cleanup-trace-events.pl trace-events | diff trace-events" is
now clean again.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit ddd0bd480fc07cc45f9cc7e3d113f23cb58b6082)

Second hunk dropped, because we don't have commit 42e5b4c.
---
 trace-events | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 trace-events | 1 -
 1 file changed, 1 deletion(-)

diff --git a/trace-events b/trace-events
index 2a034cb..2336943 100644
--- a/trace-events
+++ b/trace-events
@@ -1102,7 +1102,6 @@ qemu_spice_wakeup(uint32_t qid) "%d"
 qemu_spice_create_update(uint32_t left, uint32_t right, uint32_t top, uint32_t bottom) "lr %d -> %d,  tb -> %d -> %d"
 
 # hw/display/qxl-render.c
-qxl_render_blit_guest_primary_initialized(void) ""
 qxl_render_blit(int32_t stride, int32_t left, int32_t right, int32_t top, int32_t bottom) "stride=%d [%d, %d, %d, %d]"
 qxl_render_guest_primary_resized(int32_t width, int32_t height, int32_t stride, int32_t bytes_pp, int32_t bits_pp) "%dx%d, stride %d, bpp %d, depth %d"
 qxl_render_update_area_done(void *cookie) "%p"
-- 
1.7.11.7

